### PR TITLE
Fill in `errors` for hop nodes when Last Seen is out of date, and clear them when not

### PR DESCRIPTION
**Pull request by AlanCoding**
Connect https://github.com/ansible/awx/issues/11532

### CODE REVIEW

**Comment by AlanCoding** [comment]
From discussions I had, it sounds like this is doing the expected thing.

File: awx/main/utils/common.py [hunk 1]
```diff
@@ -756,6 +756,9 @@ def get_mem_effective_capacity(mem_mb):
 
 
 def get_corrected_memory(memory):
+    if memory == 0:
+        return memory
```
Comment (AlanCoding) [inline awx/main/utils/common.py]: This might go better in `get_mem_effective_capacity`

**Review by jbradberry — APPROVED** [review]

**Comment by AlanCoding** [comment]
I relaunched some test results and they all look pretty good.

File: awx/main/tasks/system.py [hunk 2]
```diff
@@ -458,11 +457,11 @@ def inspect_execution_nodes(instance_list):
 
             # Only execution nodes should be dealt with by execution_node_health_check
             if instance.node_type == 'hop':
+                logger.warn(f'Hop node {hostname}, has rejoined the receptor mesh')
```
Comment (jbradberry) [inline awx/main/tasks/system.py]: The `.warn()` method is deprecated and throws a warning.  Please use `.warning()` instead.

Comment (AlanCoding) [reply awx/main/tasks/system.py]: d'oh, I should have known better.

**Pull request merged**